src/main/java/it/pagopa/selfcare/pagopa/backoffice/service/ApiManagementService.java

```diff
@@ -61,14 +61,23 @@ public ApiManagementService(AzureApiManagerClient apimClient,
         this.environment = environment;
     }
 
-    public List<InstitutionDetail> getInstitutions() {
-        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
-        Collection<InstitutionInfo> institutions = externalApiClient.getInstitutions(Utility.extractUserIdFromAuth(authentication));
-        return institutions.stream()
-                .map(institution -> modelMapper.map(institution, InstitutionDetail.class))
-                .toList();
+    public List<InstitutionDetail> getInstitutions(String taxCode) {
+        if(taxCode != null && !taxCode.isEmpty()) {
+            return externalApiClient.getInstitutionsFiltered(taxCode).getInstitutions().stream()
```

Comment:
any user with a valid JWT can invoke this API, should we check if the user is an operator?